### ISSUE TITLE
No confimation after Delete, submit, etc

### ISSUE THREAD

**Issue by TomDaria** [umbraco/Umbraco-CMS#5975]
We are on Umbraco v7.11.1
When you delete something (or submit for deletion), we get no user feedback (i.e. "Item has been deleted").

Is this fixed in a later version of Umbraco.  If not, can you fix this?

Thanks

Tom

**Comment by kjac** [comment]
@TomDaria by "something" do you mean content items, media items, ...?

**Comment by TomDaria** [comment]
content items.  After you click OK, the screen blinks but there is no system User Interface confirmation that it deleted.  Our workaround is to look in the recycle bin.

**Comment by kjac** [comment]
@TomDaria here's a GIF of 7.15 in action:

![delete-content-node](https://user-images.githubusercontent.com/7405322/61770432-82832d00-aded-11e9-8f3c-a684eacca35c.gif)

**Comment by nul800sebastiaan** [comment]
I guess @TomDaria is suggesting it would be nice to show a green banner at the bottom saying "Article one has been sent to the recycle bin"?

**Comment by TomDaria** [comment]
Yes

Sent from Yahoo Mail on Android 
 
  On Mon, Jul 22, 2019 at 1:53 PM, Kenn Jacobsen<notifications@github.com> wrote:   
@TomDaria by "something" do you mean content items, media items, ...?

**Comment by TomDaria** [comment]
Thanks for the video but that is the same as 7.11.1.
It would be great if there was some notification on like the bottom of screen, "item has been sent to recycle bin"  or something like that.
This would be for anything deleted.

Sent from Yahoo Mail on Android 
 
  On Wed, Jul 24, 2019 at 2:32 AM, Kenn Jacobsen<notifications@github.com> wrote:   
@TomDaria here's a GIF of 7.15 in action:

**Comment by TomDaria** [comment]
It would be great if there was some notification on like the bottom of screen, "item has been sent to recycle bin"  or something like that.
This would be for anything deleted (any media item and any content.
Thanks
Tom
Sent from Yahoo Mail on Android 
 
  On Wed, Jul 24, 2019 at 9:08 AM, Sebastiaan Janssen<notifications@github.com> wrote:   
I guess @TomDaria is suggesting it would be nice to show a green banner at the bottom saying "Article one has been sent to the recycle bin"?

**Comment by kjac** [comment]
If I'm not entirely mistaken, tree actions that directly affect a node are confirmed in the action context menu like this:

![image](https://user-images.githubusercontent.com/7405322/61927914-ed13a480-af76-11e9-9cb6-a8c486e0863a.png)

..or this:

![image](https://user-images.githubusercontent.com/7405322/61927979-1d5b4300-af77-11e9-9ebe-c8790faf1c39.png)

That applies to copy, move, restore, permissions, public access and probably more.

Point is... tree actions do not use the green pop-up banner, far as I can recall. The green banner is however used for bulk operations in list views.

Long story short: @nul800sebastiaan would you like me to implement the context menu confirmation instead? This would be for both V7 and V8 as it lacks in both versions.

**Comment by kjac** [comment]
PRs in #6004 (for V7) and #6005 (for V8)

**Comment by nul800sebastiaan** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/pull/6004 and https://github.com/umbraco/Umbraco-CMS/pull/6005

Cherry picked for 7.15.3: c6e32019c618df66b7760478f2977b913fbca383
Cherry picked for 8.1.4: c7cb3f81f4da891898cf009c6bf00ea08fedd562
